src/features/commands/book/book.js

```diff
@@ -0,0 +1,64 @@
+const {
+  RoomField,
+  HourField,
+  DateField,
+} = require('../../components/fields/index')
+const { ROOMS } = require('../../../utils/config')
+module.exports = async function(bot, message) {
+  const trigger_id = message.trigger_id
+  const messageStr = JSON.stringify({
```

Comment:
What do you use this for?

Reply:
Tricks of new modals to being able to reply to a success message